Examples/MAX32665/BLE_LR_Central/main.c

```diff
@@ -0,0 +1,292 @@
+/*************************************************************************************************/
+/*!
+ * @file    main.c
+ * @brief   Long range demo on a central device
+*
+*  Copyright (c) 2013-2019 Arm Ltd. All Rights Reserved.
+*
+*  Copyright (c) 2019 Packetcraft, Inc.
+*
+*  Partial Copyright (c) 2023 Analog Devices, Inc.
+*
+*  Licensed under the Apache License, Version 2.0 (the "License");
+*  you may not use this file except in compliance with the License.
+*  You may obtain a copy of the License at
+*
+*      http://www.apache.org/licenses/LICENSE-2.0
+*
+*  Unless required by applicable law or agreed to in writing, software
+*  distributed under the License is distributed on an "AS IS" BASIS,
+*  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+*  See the License for the specific language governing permissions and
+*  limitations under the License.
+*/
+/*************************************************************************************************/
+
+#include <string.h>
+#include "wsf_types.h"
+#include "wsf_trace.h"
+#include "wsf_bufio.h"
+#include "wsf_msg.h"
+#include "wsf_assert.h"
+#include "wsf_buf.h"
+#include "wsf_heap.h"
+#include "wsf_cs.h"
+#include "wsf_timer.h"
+#include "wsf_os.h"
+
+#include "sec_api.h"
+#include "hci_handler.h"
+#include "dm_handler.h"
+#include "l2c_handler.h"
+#include "att_handler.h"
+#include "smp_handler.h"
+#include "l2c_api.h"
+#include "att_api.h"
+#include "smp_api.h"
+#include "app_api.h"
+#include "hci_core.h"
+#include "app_terminal.h"
+#include "wut.h"
+#include "rtc.h"
+#include "trimsir_regs.h"
+
+#if defined(HCI_TR_EXACTLE) && (HCI_TR_EXACTLE == 1)
+#include "ll_init_api.h"
+#endif
+
+#include "pal_bb.h"
+#include "pal_cfg.h"
+#include "pal_timer.h"
+#include "pal_sys.h"
+
+#include "lr_central_api.h"
+#include "app_ui.h"
+
+/**************************************************************************************************
+  Macros
+**************************************************************************************************/
+
+/*! \brief UART TX buffer size */
+#define PLATFORM_UART_TERMINAL_BUFFER_SIZE 2048U
+#define DEFAULT_TX_POWER 0 /* dBm */
```

Comment:
Default slave TX power is 4dBm, should default master TX power be 4 dBm as well for larger connection range?